src/main/java/io/jenkins/plugins/opentelemetry/backend/elastic/ElasticsearchLogStorageRetriever-v7.txt

```diff
@@ -0,0 +1,303 @@
+/*
```

Comment:
a txt file? looks like a leftover

Reply:
kept here to be prepared to rollback or compare if needed.